v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
That wording sounds find to me! Copying my comment from below since I couldn't comment here the other day for some reason:

>"1-indexed" means the array indices start from 1, rather than 0, (so (ARRAY[1,2,3])[1] is 1, not 2), and homogenous does indeed mean that all values have the same type. These are both the technical terms for these concepts but I certainly wouldn't object to more accessible terminology!